appinventor/appengine/src/com/google/appinventor/client/youngandroid/YoungAndroidFormUpgrader.java

```diff
@@ -1140,6 +1140,13 @@ private static int upgradeLocationSensorProperties(Map<String, JSONValue> compon
       // The speed parameter was added to the LocationChanged event
       srcCompVersion = 3;
     }
+    if (srcCompVersion < 4) {
+      // Make location sensor disible by default
+      if (!componentProperties.containsKey("Enabled")) {
+        componentProperties.put("Enabled", new ClientJsonString("True"));
```

Comment:
As I see currently, upgrader will set default Enabled property's value to `true`, instead of `false`.

Reply:
Yes, this is keeping the default value true for the old projects.